about.html

```diff
@@ -0,0 +1,162 @@
+            CREATIVELY START YOUR <br/>INTERIOR DESIGN LEARNING CAREER
+          </h1>
+        </div>
+        <div class="modal">
+          <p class="modal-desktop">
+           Graduates of the Cozy Space Interior Design are prepared to begin 
+          careers or pursue further education in interior design, lighting<br/>
+           design, sustainable design, set design, exhibition and event design,
+            historic preservation, and consulting.
+          </p>
+          <p class="modal-mobile">
+            Graduates of the Cozy Space Interior Design are prepared to begin 
+          careers or pursue further education in interior design, lighting<br/>
+           design, sustainable design, set design, exhibition and event design,
+            historic preservation, and consulting.
+          </p>
+        </div>
+        <div class="contact-me">
+          <p>
+            Please contact us per Email for any further questions Cozy Space celebration 2022!
+          </p>
+          <a href="kdan2944@gmail.com"
+            >kdan2944@gmail.com <i class="fas fa-angle-right"></i
+          ></a>
+        </div>
+      </section>
+      <section class="global">
+        <h1 class="global-header">
+          About the company 
+        </h1>
+        <hr class="hr" />
+        <p class="global-text">
+          Offering design programs  than any other interior designing schools,
+           Cosy Space is uniquely qualified to prepare talented students for professional, creative careers
+        </p>
+        <div class="img-globe">
+          <img
+            src="./images/cozy-mob2.png"
+            class="global-img"
+            alt="global"
+          />
+        </div>
+      </section>
+      <section class="alumni">
+        <h2>See CozySpace history</h2>
+        <hr class="hr">
+        <p>
+          The Cozy Space school of interior Design was founded in 2018 by Richard G. 
+           to provide designing carer programs in-person and online as well
+           not previously available in southeast Georgia and to create a specialized professional art school to attract 
+           students from throughout the United States and abroad.
+        </p>
+        <div class="alumni-con">
+          <div class="alumni-img">
+            <h2>2021</h2>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsum,
+              reprehenderit.
+            </p>
+          </div>
+          <div class="alumni-img">
+            <h2>2022</h2>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsum,
+              reprehenderit.
+            </p>
+          </div>
+        </div>
+      </section><section class="partner">
+      <h2>Partners</h2>
+      <hr class="hr" />
+      <div class="partner-logo">
+          <p>Gensler</p>
+          <p> HBA</p>
+          <p>yabu pushelberg</p>
+          <p>INTERIOR ARCHITECTS</p>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly as I asked in the first review you don't need this partner section in the about page for the desktop version check well how the design and you will notice that ✅👍🏻👍🏻